include/igl/readCSV.cpp

```diff
@@ -30,8 +30,10 @@ IGL_INLINE bool igl::readCSV(
     std::istringstream iss(line);
     vector<Scalar> temp;
     Scalar a;
-    while (iss >> a)
+    char ch;
+    while (iss >> a >> ch)
       temp.push_back(a);
+    temp.push_back(a);
```

Comment:
This will not work if the csv is empty.

Reply:
Thank you. I have changed line 36 to
`if(iss >> a)
    temp.push_back(a);
`
and merged into the same commit. 

Do I have to make another pull request?

Reply:
No just update your branch and you'll be fine. But I think your fix still doesn't cover all cases. I would change it to the following:
```cpp
while (iss >> a) {
  temp.push_back(a);
  if (!(iss >> ch)) {
    break;
  }
}
```

Reply:
Ya, you are right again. I will fix it.

Reply:
I fixed it. You can pull now.